openslides/core/views.py

```diff
@@ -213,3 +214,85 @@ class CustomSlideDeleteView(CustomSlideViewMixin, utils_views.DeleteView):
     Delete a custom slide.
     """
     pass
+
+
+class TagException(Exception):
+    pass
+
+
+class TagListView(utils_views.AjaxMixin, utils_views.ListView):
+    """
+    View to list and manipulate tags.
+
+    Shows all tags when requested via a GET-request. Manipulates tags with
+    POST-requests.
+    """
+
+    model = Tag
+    required_permission = 'core.can_manage_tags'
+
+    def post(self, *args, **kwargs):
+        return self.ajax_get(*args, **kwargs)
+
+    def ajax_get(self, request, *args, **kwargs):
+        name, value = request.POST['name'], request.POST.get('value', None)
+
+        # Create a new tag
+        if name == 'new':
+            try:
+                tag = Tag.objects.create(name=value)
+            except IntegrityError:
+                # The tag-name is already taken (it must be uniquei)
```

Comment:
Typo: ~~# The tag-name is already taken (it must be uniquei)~~ # The name of the tag is already taken. It must be unique.